packages/core/lib/commands/test.js

```diff
@@ -162,11 +104,83 @@ const command = {
         (started, disconnect) => {
           ipcDisconnect = disconnect;
           Environment.develop(config, ganacheOptions)
-            .then(() => environmentCallback())
+            .then(() => copyArtifactsToTempDir(config))
+            .then(({ config, temporaryDirectory }) => {
+              return prepareConfigAndRunTests({
+                config,
+                files,
+                temporaryDirectory
+              });
+            })
+            .then(numberOfFailures => {
+              done.call(null, numberOfFailures);
+              ipcDisconnect();
+            })
             .catch(done);
         }
       );
     }
+  },
+
+  helpers: {
```

Comment:
Why not just move these to `lib/test/`? That way you can have the imports at the top of the file, all in one place, like one might expect.

Reply:
What about moving all test stuff into a `test` folder? I've wanted to start this pattern in `core`. So instead of having a folder with all the command files, you would have a bunch of directories with directories for each command. I would like to save `lib` for stuff that is used by more than one command. How does that seem?